process/block/shardblock.go

```diff
@@ -534,12 +533,12 @@ func (sp *shardProcessor) restoreMetaBlockIntoPool(mapMiniBlockHashes map[string
 
 	for metaBlockHash, miniBlockHashes := range mapMetaHashMiniBlockHashes {
 		for _, miniBlockHash := range miniBlockHashes {
-			sp.addProcessedMiniBlock([]byte(metaBlockHash), miniBlockHash)
+			sp.processedMiniBlocks.AddMiniBlockHash([]byte(metaBlockHash), miniBlockHash)
```

Comment:
why are you casting to byte array and in the structure you cast back to string ?

Reply:
done. now string is used.